packages/agent/conf/demos/dbc/oid4vci_metadata/dbc.json

```diff
@@ -59,39 +59,39 @@
           }
         ],
         "order": [
-          "Voornaam",
-          "Achternaam",
+          "firstName",
+          "lastName",
           "email"
         ],
         "credentialSubject": {
-          "Voornaam": {
+          "firstName": {
             "value_type": "string",
             "display": [
               {
-                "name": "FirstName"
+                "name": "First Name"
               },
               {
-                "name": "FirstName",
+                "name": "First Name",
                 "locale": "en-US"
               },
               {
-                "name": "Voornaam",
+                "name": "firstName",
                 "locale": "nl-NL"
               }
             ]
           },
-          "Achternaam": {
+          "lastName": {
             "value_type": "string",
             "display": [
               {
-                "name": "LastName"
+                "name": "Last Name"
```

Comment:
this is the default branding when there is no locale. it is a dutch conference. should we not set this to dutch?

Reply:
see prev